tests/mutating-requests-tests.bats

```diff
@@ -1,22 +1,23 @@
 #!/usr/bin/env bats
 
-source $BATS_TEST_DIRNAME/common.bash
+setup() {
+	load common.bash
+	wait_pods
+}
 
-setup_file() {
-	kubectl --context $CLUSTER_CONTEXT delete --wait --ignore-not-found pods --all
-	kubectl --context $CLUSTER_CONTEXT delete --wait --ignore-not-found -n kubewarden clusteradmissionpolicies --all
-	kubectl --context $CLUSTER_CONTEXT wait --for=condition=Ready -n kubewarden pod --all
+teardown_file() {
+	kubectl delete --wait --ignore-not-found pods --all
+	kubectl delete --wait --ignore-not-found -n kubewarden clusteradmissionpolicies --all
 }
 
 @test "[Mutation request tests] Install mutate policy with mutating flag enabled" {
 	apply_cluster_admission_policy $RESOURCES_DIR/mutate-policy-with-flag-enabled.yaml
 }
 
 @test "[Mutation request tests] Launch a pod that should be mutate by psp-user-group-policy" {
-	kubectl_apply_should_succeed $RESOURCES_DIR/mutate-pod-psp-user-group-policy.yaml
-	run kubectl --context $CLUSTER_CONTEXT wait --for=condition=Ready pod pause-user-group
-	run eval `kubectl --context $CLUSTER_CONTEXT get pod pause-user-group -o json | jq ".spec.containers[].securityContext.runAsUser==1000"`
-	[ "$status" -eq 0 ]
```

Comment:
shouldn't we still be checking the retcode after the kubectl get pod?

Reply:
This code will never fail, jq always returns 0 if you use valid syntax.
`runAsUser==1000` returns string (true|false), but exit code is always 0.

I replaced it with `jq -e` to set return code from condition, there is no need to user `run` wrapper here.

Reply:
thanks!